invesalius/data/log.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
```

Comment:
I think it's better to move this file to invesalius/gui/

Reply:
Will do.

Reply:
Done